### PR TITLE
Products and Wishlist

### PR DESCRIPTION
**Pull request by sudipkundu999**
### Please review only `.jsx` files.
All css files have been reviewed earlier
I'm using fake data till I integrate Mockbee

Preview of the Products page
Screenshot (73)

Preview of Wishlist page
Screenshot (74)

### CODE REVIEW

File: src/pages/Wishlist.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import "./pages-css/wishlist.css";
+import { data } from "../FAKE.DATA";
+import { ProductCard } from "../components";
+export const Wishlist = () => {
+  return (
+    <main className="wishlist-main">
+      <div className="heading">Wishlist</div>
+      <div className="cards-wrapper">
+        {data.map((x) => (
```
Comment (humzahasan) [inline src/pages/Wishlist.jsx:9]: Can give more relevant name than 'x' and more specific than 'data'

**Review by humzahasan — APPROVED** [review]
UI Suggestion : Your card in Cart can have a bit more of margin and padding. The image can have a top margin as well as bottom margin. 

Code changes look good.  You can implement changes when refactoring again with mockbee.

File: src/components/ProductFilter.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            Self Help
+          </label>
+        </div>
+        <div className="filter-category-wrapper">
+          <label>
+            <input type="checkbox" name="category" />
+            Mystery
+          </label>
+        </div>
+      </div>
```
Comment (iRohitGaur) [inline src/components/ProductFilter.jsx:57]: suggestion: you can map this from an array

**Review by iRohitGaur — APPROVED** [review]
UI looks nice. Good to merge 👍🏻

File: src/pages/Products.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { data } from "../FAKE.DATA";
+import "./pages-css/products.css";
+
+export const Products = () => {
+  return (
+    <div className="products-body">
+      <ProductFilter />
+
+      <main className="products-main">
+        {data.map((x) => (
```
Comment (archit7448) [inline src/pages/Products.jsx:12]: rather than x you can give meaning full name

**Review by archit7448 — APPROVED** [review]
looks good to merge Nice UI

File: src/pages/Wishlist.jsx (hunk 1 above)
Comment (archit7448) [reply src/pages/Wishlist.jsx:9]: +1

Comment (sudipkundu999) [reply src/pages/Wishlist.jsx:9]: Ok will take care of this from subsequent PRs

File: src/components/ProductFilter.jsx (hunk 2 above)
Comment (sudipkundu999) [reply src/components/ProductFilter.jsx:57]: Noted. I will take care of this in the subsequent PRs

**Pull request merged**